Doc/whatsnew/3.11.rst

```diff
@@ -62,6 +62,7 @@ Summary -- Release highlights
 .. This section singles out the most important changes in Python 3.11.
    Brevity is key.
 
+- Python 3.11 is 22% faster than Python 3.10. See `Faster CPython`_ for details.
```

Comment:
I would rather claim a range, say 10% to 60%.

Once the latest PRs are merged it will be about 25%, if you must have a single number.

https://gist.github.com/markshannon/bf1b99177290db30dd0ab8f56446d0ea

Reply:
I will update this number when the benchmark link updates its values/ when those PRs are merged into `main`.